### ISSUE TITLE
Compile error in opm-grid

### ISSUE THREAD

**Issue by joakim-hove** [OPM/opm-grid#359]
I am on a stock Ubuntu 16.04: 
```
In file included from /home/hove/work/OPM/opm-grid/tests/cpgrid/grid_nnc.cpp:31:0:
/home/hove/work/OPM/opm-grid/opm/grid/CpGrid.hpp:1329:16: warning: 'template<class Grid> struct Dune::Capabilities::isParallel' is deprecated [-Wdeprecated-declarations]
         struct isParallel<CpGrid>
                ^
In file included from /home/hove/work/OPM/opm-grid/opm/grid/CpGrid.hpp:56:0,
                 from /home/hove/work/OPM/opm-grid/tests/cpgrid/grid_nnc.cpp:31:
/usr/include/dune/grid/common/capabilities.hh:72:5: note: declared here
     isParallel
     ^
/home/hove/work/OPM/opm-grid/tests/cpgrid/grid_nnc.cpp: In member function 'void Fixture::testCase(const string&, const Opm::NNC&, int, int, int, const std::vector<std::pair<int, int> >&, bool)':
/home/hove/work/OPM/opm-grid/tests/cpgrid/grid_nnc.cpp:79:44: error: 'elements' was not declared in this scope
         for (const auto& elem : elements(gv)) {
                                            ^
/home/hove/work/OPM/opm-grid/tests/cpgrid/grid_nnc.cpp:81:60: error: 'intersections' was not declared in this scope
             for (const auto& inter : intersections(gv, elem)) {
                                                            ^
CMakeFiles/grid_nnc.dir/build.make:62: recipe for target 'CMakeFiles/grid_nnc.dir/tests/cpgrid/grid_nnc.cpp.o' failed
make[2]: *** [CMakeFiles/grid_nnc.dir/tests/cpgrid/grid_nnc.cpp.o] Error 1
CMakeFiles/Makefile2:136: recipe for target 'CMakeFiles/grid_nnc.dir/all' failed
make[1]: *** [CMakeFiles/grid_nnc.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
```
must admit that building with MPI is not my strongest point - might be a triviality?

**Comment by andlaus** [comment]
cf #358

**Comment by joakim-hove** [comment]
> cf #358

Ahh - thanks; I thought that was merged.

**Comment by blattms** [comment]
Fixed with the merge of #358
